modules/protocol/nefarious.c

```diff
@@ -365,10 +365,18 @@ static void m_nick(sourceinfo_t *si, int parc, char *parv[])
 	{
 		/* -> AB N jilles 1 1137687480 jilles jaguar.test +oiwgrx jilles B]AAAB ABAAE :Jilles Tjoelker */
 		/* -> AB N test4 1 1137690148 jilles jaguar.test +iw B]AAAB ABAAG :Jilles Tjoelker */
-		slog(LG_DEBUG, "m_nick(): new user on `%s': %s", si->s->name, parv[0]);
+
+		/*
+		 * -> Mh N jilles 1 1460435852 jilles 127.0.0.1 +ixCc 1A130C.572B1.6F53B5.8DD3B8.IP 1A130C.572B1.6F53B5.8DD3B8.IP DSBHPW Mhw2O :Real Name
```

Comment:
Replace old variants with this comment.  Maybe document the change like so "this was changed in Nefarious 2."